internal/service/account/region.go

```diff
@@ -239,3 +244,10 @@ func waitRegionDisabled(ctx context.Context, conn *account.Client, accountID, re
 
 	return nil, err
 }
+
+func requiresStatusChange(status types.RegionOptStatus, enable bool) bool {
+	if enable {
+		return status != types.RegionOptStatusEnabled
```

Comment:
To ensure we also skip when the region is actively being enabled.

```suggestion
		return status != types.RegionOptStatusEnabled && status != types.RegionOptStatusEnabling
```